test/server/parser/parser.ts

```diff
@@ -0,0 +1,75 @@
+
```

Comment:
Did you mean to miss the license and authorship information from the top of this file?

Reply:
That was a mistake on my part. Updating the PR!